staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -246,9 +250,6 @@ func processJSONPathInput(jsonPathExpression, jsonPathCond string) (string, stri
 	if err != nil {
 		return "", "", err
 	}
-	if jsonPathCond == "" {
-		return "", "", errors.New("jsonpath wait condition cannot be empty")
-	}
```

Comment:
Removing this code introduces a bug when the command is called in the form of `--for=jsonpath='{.status.readyReplicas}'=` i.e. without the value after second `=` sign.

Reply:
Handled this case